docs/api/browser-window.md

```diff
@@ -662,6 +668,13 @@ Emitted when details regarding a requested resource is available.
 
 Emitted when a redirect was received while requesting a resource.
 
+### Event: 'page-favicon-updated'
+
+* `event` Event
+* `favicons` [String]
```

Comment:
We usually use `Array` instead of `[...]` for parameter type, `[...]` makes the parameter look optional.